setup.py

```diff
@@ -1,22 +1,44 @@
-from setuptools import setup
+from setuptools import setup, find_packages
+import sys
 
```

Comment:
Can we put this one in `ubuntu/setup.py`?

Reply:
Could you please test the app on the mac with current changes?
I believe the structure must be kept so that the snaps works as expected but I don't think the current changes pose a problem to mac installation. Can you confirm that?